tritium-registry/src/main/java/com/palantir/tritium/metrics/registry/DefaultTaggedMetricRegistry.java

```diff
@@ -22,18 +22,18 @@
 @AutoService(TaggedMetricRegistry.class)
 public final class DefaultTaggedMetricRegistry extends AbstractTaggedMetricRegistry {
 
-    private static final TaggedMetricRegistry DEFAULT = new DefaultTaggedMetricRegistry();
-
     public DefaultTaggedMetricRegistry() {
         super(ExponentiallyDecayingReservoir::new);
     }
 
     /**
      * Get the global default {@link TaggedMetricRegistry}.
+     * @deprecated inline this method
```

Comment:
nit: link the replacement
```suggestion
     * @deprecated use SharedTaggedMetricRegistries#getDefault
```